Source/CarthageKit/FrameworkExtensions.swift

```diff
@@ -275,11 +275,11 @@ extension URL {
 			throw error(failureReason: "Expected single resource value: «actual count: \(values.count)».")
```

Comment:
Though I'm not sure how this would happen event if we only pass single key to `resourceValues(forKeys:)` (have you ever experienced such unexpected behavior?), let's keep it for now.